### DISCUSSION TITLE
Add ability to create sql snippets on selfhosted instance

### DISCUSSION THREAD

**Discussion by Chiggy-Playz** [supabase/supabase#13774] [Feature Requests]
# Bug report

## Describe the bug
There's no getting started or new query button to create a new sql snippet. 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Selfhost supabase
2. Open on SQL Editor
3. There's no new query button

## Expected behavior

Expected to see new query button as well as get started to quickly create snippets
## Screenshots

Example (first image is supabase cloud, second  is selfhosted using docker): 
![image](https://user-images.githubusercontent.com/49774426/222844982-3ad060dd-015e-401c-b3d7-d640ffca34c8.png)
![image](https://user-images.githubusercontent.com/49774426/222844991-c8c6aaf4-9d5e-4e62-93b2-914ff3b468ea.png)

## System information

- OS: Debian 11.6
- Browser Edge
- Selfhosted via docker
- Studio version: 20230228-029c7bd

**Comment by GaryAustin1** [comment]
This discussion comments that SQL snippets are not available on self-hosted.
https://github.com/supabase/supabase/discussions/12388
I'm marking this as an enhancement request

**Comment by MildTomato** [comment]
I'm going to move this to discussions feature request 🚀

**Comment by MoradFN** [comment]
So, what happened?
